AEPExperiencePlatform.xcodeproj/project.pbxproj

```diff
@@ -1501,6 +1507,8 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.adobe.mobile.AEPCommerceDemoApp;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_OBJC_BRIDGING_HEADER = "SampleApps/AEPCommerceDemoApp/AEPCommerceDemoApp/AEPCommerceDemoApp-Bridging-Header.h";
```

Comment:
This should not be required anymore

Reply:
I attempted to remove it, however, importing `AEPAssurance` still fails.

Reply:
nevermind, had to remove all the pods and re-add. working now.